src/statisticBlock.js

```diff
@@ -468,8 +470,14 @@ class StatisticBlock extends Component {
 
   onDimensionLabelClick(dimNo, value) {
     const { services } = this.props;
+    const component = services.QlikComponent;
+    const inStoryMode = component.options.selections;
     if (services && services.QlikComponent) {
-      services.QlikComponent.selectValues(dimNo, [value], true);
+      if(inStoryMode == false){
+        return;
+      }else {
+        services.QlikComponent.selectValues(dimNo, [value], true);
+      }
```

Comment:
This section could use some more love and simplification. I think this would look more straightforward:
```js
  onDimensionLabelClick(dimensionNumber, value) {
    const { services } = this.props;
    if (!services || !services.QlikComponent || !services.QlikComponent.options) {
      return;
    }
    const inStoryMode = services.QlikComponent.options.selections;
    if (inStoryMode) {
      services.QlikComponent.selectValues(dimensionNumber, [value], true);
    }
  }
```

Let me know what you think